handle_actions.go

```diff
@@ -70,54 +78,63 @@ func (a ActionReplyHTTP) Perform(context Context) error {
 	for k, v := range a.Headers {
 		ec.Response().Header().Set(k, v)
 	}
+
 	msg, err := context.Render(a.Body)
 	if err != nil {
 		logrus.WithField("err", err).Error("failed to render template for http")
-		return err
 	}
-	return ec.Blob(a.StatusCode, contentType, []byte(msg))
+
+	err = ec.Blob(a.StatusCode, contentType, []byte(msg))
+	if err != nil {
+		handleActionErr(a, err)
+	}
 }
 
-func (a ActionRedis) Perform(context Context) error {
+// Perform defines the action to perform for the ActionRedis action
+func (a ActionRedis) Perform(context Context) {
 	for _, cmd := range a {
 		_, err := context.Render(cmd)
 		if err != nil {
-			return err
+			handleActionErr(a, err)
 		}
 	}
-	return nil
 }
 
-func (a ActionSleep) Perform(context Context) error {
+// Perform defines the action to perform for the ActionSleep action
+func (a ActionSleep) Perform(context Context) {
 	time.Sleep(a.Duration)
-	return nil
 }
 
-func (a ActionPublishKafka) Perform(context Context) error {
+// Perform defines the action to perform for the ActionPublishKafka action
+func (a ActionPublishKafka) Perform(context Context) {
 	msg := a.Payload
 	msg, err := context.Render(msg)
 	if err != nil {
 		logrus.WithField("err", err).Error("failed to render template for kafka payload")
-		return err
```

Comment:
we will still log an error but we won't _also_ log an action error. seems ok to me since errors are descriptive but curious to know what others think.